CodeEdit/Features/Tasks/Models/CEActiveTask.swift

```diff
@@ -72,11 +80,12 @@ class CEActiveTask: ObservableObject, Identifiable, Hashable {
 
     func handleProcessFinished(terminationStatus: Int32) async {
         handleTerminationStatus(terminationStatus)
+        print(terminationStatus)
```

Comment:
Probably should be removed. Could be replaced with an actual log

Reply:
Replaced with logger